microsoft-edge/devtools-guide-chromium/javascript/snippets.md

```diff
@@ -22,149 +22,158 @@ ms.date: 05/04/2021
    limitations under the License.  -->
 # Run snippets of JavaScript on any webpage
 
-If you are running the same code in the [Console](../console/index.md) repeatedly, consider saving the code as a Snippet instead.  Snippets are scripts that you author in the [Sources](../sources/index.md) tool.  Snippets have access to the JavaScript context of the webpage, and you can run Snippets on any webpage.  The security settings of most webpages block from loading other scripts in Snippets.  For that reason, you must include all your code in one file.
+If you're running the same code in the **Console** tool repeatedly, consider saving the code as a snippet instead.  Snippets are scripts that you author in the **Sources** tool.  Snippets have access to the JavaScript context of the webpage and you can run snippets on any webpage.  Snippets can be used to alter a webpage, for example to change its content or appearance, or to extract data.
```

Comment:
```suggestion
If you're entering the same code into the **Console** tool repeatedly, consider saving the code as a snippet instead, and then running the snippet.  Snippets are scripts that you author in the **Sources** tool.  Snippets have access to the JavaScript context of the webpage and you can run snippets on any webpage.  Snippets can be used to alter a webpage, for example to change its content or appearance, or to extract data.
```
* "entering" - this is a more active, direct verb describing the labor.  "running" sounds more automatic/passive/abstract.
* ", and then running the snippet" - a more explicit, complete story of what we're proposing.